UnleashClient/loader.py

```diff
@@ -62,7 +62,9 @@ def _create_feature(provisioning: dict,
     return Feature(name=provisioning["name"],
                    enabled=provisioning["enabled"],
                    strategies=parsed_strategies,
-                   variants=variant
+                   variants=variant,
+                   impression_data=provisioning["impressionData"],
+                   stale=provisioning["stale"]
```

Comment:
Looks like these will fail to load if these properties aren't present, which I'd rather avoid. `impression_data` won't be there prior to Unleash 4.7 and I'm not certain we can guarantee either of them in the bootstrap files. Can we swap this out for something more tolerant to these properties being absent like `provisioning.get("impressionData")`?

Reply:
Sure, I can make it fail-safe. I didn't know such details, so I thought that should be safe.

Reply:
this is done.